### PR TITLE
Bump NodeJS in workflow runs

### PR DESCRIPTION
**Pull request by queengooborg**
This PR bumps the NodeJS version used in workflow runs up to NodeJS v18, the now-current version of Node.

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-node@v2
       with:
-        node-version: 16.x
+        node-version: 18.x
```
Comment (caugner) [inline .github/workflows/node.js.yml:20]: The `.x` does not seem to be necessary (see [here](https://github.com/actions/setup-node#supported-version-syntax)):

```suggestion
        node-version: 18
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-node@v2
       with:
-        node-version: 16.x
+        node-version: 18.x
```
Comment (caugner) [inline .github/workflows/release.yml:19]:
```suggestion
        node-version: 18
```

**Comment by queengooborg** [comment]
I've now set up a testing matrix to test our code in all LTS versions of NodeJS!

**Comment by queengooborg** [comment]
We could probably set up a matrix to test multiple NodeJS versions, though I'm not able to modify what checks are required so I can't really make that change here.  😛

**Pull request merged**